packages/core/src/exchange/index.ts

```diff
@@ -0,0 +1,29 @@
+import type { Signer } from 'ethers'
+import type { Sdk } from '../graphql'
+import type { UUID } from '../types'
+import * as ListToken from './listToken'
+import * as PlaceBid from './placeBid'
+
+export class Exchange {
+  private readonly sdk: Sdk
+
+  constructor(sdk: Sdk) {
+    this.sdk = sdk
+  }
+
+  placeBid(
+    bid: PlaceBid.Bid,
+    signer: Signer,
+    onProgress?: (action: PlaceBid.Action) => void,
+  ): Promise<UUID> {
+    return PlaceBid.placeBid(this.sdk, bid, signer, onProgress)
```

Comment:
Could you explain the reason to have the Client and Exchange as class but the rest as simple function? I will not be against having everything as class.

Reply:
I initially started with all functions and then grouped them into class. 
I would love to keep pure-ish functions as they would be easier to test (and I do plan to add the tests), and it's great to have a dedicated file for each action for code organization, but I'm not against putting all in the class directly. That was a kind question mark for now

Reply:
Here is a version with all of the classes https://github.com/liteflow-labs/liteflow-js/pull/143
A lot heavier in my opinion and quite hard to navigate

Reply:
Let's go with normal functions as it is here